test-cases/longevity/longevity-10gb-3h.yaml

```diff
@@ -1,19 +1,20 @@
-test_duration: 240
-stress_cmd: ["cassandra-stress write cl=QUORUM duration=180m -schema 'replication(factor=3) compaction(strategy=SizeTieredCompactionStrategy)' -port jmx=6868 -mode cql3 native -rate threads=1000 -pop seq=1..10000000 -log interval=5"
+test_duration: 60
```

Comment:
Basicailly you need to add a configuration that adds only the db_nodes_shards_selection parameter and override the user_prefix.

Reply:
I didn't prepare the test yet, just infrastructure to support this feature